packages/atomic/src/components/atomic-result-list/replace-breakpoint.ts

```diff
@@ -0,0 +1,47 @@
+function replaceMediaQuery(style: string, breakpoint: number) {
+  return style.replace(
+    /\(min-width: 1024px\)/g,
+    `(min-width: ${breakpoint}px)`
+  );
+}
+
+function replaceStyleSheet(element: HTMLElement, breakpoint: number) {
+  // Not adopted by all browsers, not part of Typescript yet
```

Comment:
You can augment the base `HTMLElement` interface, though. Don't need to cast as `any`.

https://basarat.gitbook.io/typescript/type-system/lib.d.ts#modifying-native-types